src/App.js

```diff
@@ -1,26 +1,139 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './component/FoodBox'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+class App extends React.Component {
+  
+state={
+  foods: foods,
+  showForm:false,
+
+  name: "",
+  calories: 0,
+  image: "",
+  quantity: 0
+
+
```

Comment:
Take care in adding too many spaces, keep your code clean also in its structure to make it more appealing and clean.